makefile.shade

```diff
@@ -19,12 +20,13 @@ default DROP_SHARE='${Environment.GetEnvironmentVariable("DROP_SHARE")}'
 default PACKAGE_DROP_SHARE='${Environment.GetEnvironmentVariable("PACKAGE_DROP_SHARE")}'
 default NUGET_SYMBOLS_SHARE='${Environment.GetEnvironmentVariable("NUGET_SYMBOLS_SHARE")}'
 default SYMBOLS_NUGET_EXE='${Environment.GetEnvironmentVariable("SYMBOLS_NUGET_EXE")}'
-default PUSH_TO_NUGET='${ !string.IsNullOrEmpty(Environment.GetEnvironmentVariable("NUGET_PUBLISH_FEED")) && !string.IsNullOrEmpty(Environment.GetEnvironmentVariable("COHERENCE_PUSH_TO_NUGET")) }'
+default NUGET_PUBLISH_FEED='${Environment.GetEnvironmentVariable("NUGET_PUBLISH_FEED")}'
+default APIKEY='${Environment.GetEnvironmentVariable("APIKEY")}'
 
 use-standard-lifecycle
 
 #compile-sanity-check target='compile'
-  build projectFile='${Path.Combine(SANITY_CHECKER, "SanityCheck.csproj")}'
+  exec program='cmd' commandline='/C dnu restore'
```

Comment:
This and [line 43](https://github.com/aspnet/Coherence/pull/34/files#diff-9c69ad97a4a478903e84088c363feebaR43) are in platform-neutral targets but will fail miserably if `IsLinux`.